NbuildTests/CommandTests.cs

```diff
@@ -176,8 +177,17 @@ public void InstallFromJsonFileTest()
 
             ResourceHelper.ExtractEmbeddedResourceFromAssembly(assembly, ResourceLocation, json);
 
+            // Replace C:\\Program Files\\Nbuild with C:\\Temp\\nbuild2
+            string jsonContent = File.ReadAllText(json);
+            jsonContent = jsonContent.Replace("$(ProgramFiles)\\\\Nbuild", "C:\\\\Temp");
```

Comment:
Modified destination of Ntools so that test does not overrive the installed version on Ntools on the local development machine.